docs/cri/installation.md

```diff
@@ -45,8 +45,6 @@ usr/local/
 usr/local/bin/
 usr/local/bin/containerd-shim-runc-v2
 usr/local/bin/ctr
-usr/local/bin/containerd-shim
-usr/local/bin/containerd-shim-runc-v1
```

Comment:
Shouldn't be removed for `cri-containerd-cni-1.4.3-linux-amd64.tar.gz`

Reply:
Should we just update the example version to 2.0.0? But seems we need versioned docs at this point

Reply:
Reverted for now.